crates/algokit_transact_ffi/src/lib.rs

```diff
@@ -390,7 +390,7 @@ impl TryFrom<AssetTransferTransactionFields> for algokit_transact::AssetTransfer
 impl TryFrom<algokit_transact::Transaction> for Transaction {
     type Error = AlgoKitTransactError;
 
-    fn try_from(tx: algokit_transact::Transaction) -> Result<Self, Self::Error> {
+    fn try_from(tx: algokit_transact::Transaction) -> Result<Self, AlgoKitTransactError> {
```

Comment:
Was this giving you a problem or is this change for clarity? I believe the `type Error = AlgoKitTransactError` on the previous line means the behavior is the same either way.